opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -1087,8 +1095,8 @@ mod tests {
                 "no metrics were exported"
             );
 
-            // Get the latest resource metric in case of multiple flushes/exports
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/pull/1766 eliminated this guess work.